common/exception/ErrorMessage.java

```diff
@@ -43,6 +43,8 @@ private ErrorMessage(String codePrefix, int codeNumber, String messagePrefix, St
 
         public static final Client CLUSTER_NOT_AVAILABLE =
                 new Client(8, "Attempted connecting to these servers, but none are available: '%s'.");
+        public static final Client SERVER_UNAVAILABLE =
+                new Client(9, "Server is unavailable.");
```

Comment:
I think "Failed to connect to server." might be more understandable for the user.

Reply:
Actually I prefer Ganesh's version.

Reply:
@alexjpwalker addressed in https://github.com/graknlabs/client-java/pull/236/commits/a25f5dfdc396650c909e336a29d6ce1cd49cc018